pkg/ansible/watches/watches.go

```diff
@@ -297,7 +294,7 @@ func getAnsibleVerbosity(gvk schema.GroupVersionKind, defValue int) int {
 	))
 	ansibleVerbosity, err := strconv.Atoi(os.Getenv(envVar))
```

Comment:
I think we should use the same logic for the ansible verbosity operator.

Since the logic is identical, I think it would make sense to define a new function with this signature:
```
func getIntegerEnvWithDefault(envVar string, defValue int) int
```
The call it for both `maxWorkers` and `ansibleVerbosity`.